main/src/main/java/cgeo/geocaching/settings/Settings.java

```diff
@@ -1047,6 +1048,10 @@ public static boolean isLiveList() {
         return getBoolean(R.string.pref_livelist, true);
     }
 
+    public static boolean isLiveDetail() {
```

Comment:
NOt a good name. People will think of "live map" which is something completely different".

How about something like "useLiveCompassInNavigationAction"?